src/database/database.cpp

```diff
@@ -7,48 +7,23 @@
 
 
 /****************
- * Black magic! *
- ****************
- * The position hashing is very messed up.
- * It's a lot more complicated than it looks.
+ * The position encoding is a bit messed up because negative
+ * values were not taken into account.
+ * But this also maps 0,0,0 to 0, which is nice, and we mostly
+ * need forward encoding in Luanti.
```

Comment:
I think this claim is dubious
```suggestion
 * But this also maps 0,0,0 to 0, which is nice.
```

Reply:
Hm, you're right.